lib/ts/framework/h3/framework.ts

```diff
@@ -0,0 +1,201 @@
+/* Copyright (c) 2021, VRAI Labs and/or its affiliates. All rights reserved.
+ *
+ * This software is licensed under the Apache License, Version 2.0 (the
+ * "License") as published by the Apache Software Foundation.
+ *
+ * You may not use this file except in compliance with the License. You may
+ * obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+
+import {
+    eventHandler,
+    send,
+    setCookie,
+    createError,
+    EventHandler,
+    H3Event,
+    sendError,
+    readBody,
+    getQuery,
+    getCookie,
+    getHeader,
+    getMethod,
+} from "h3";
```

Comment:
Our SDK cannot depend on these functions cause we can't add dependency on `h3` in our package.json (but we can in dev dependencies - which you already have). Is there a way in which these functions can be called directly on the H3Event object? For example `event.readBody()`